app/models/lunch.rb

```diff
@@ -60,9 +68,26 @@ def chalmrest
           summary, price = entry.summary.split('@')
           summary = summary.strip
           price = price.strip
-
+          if restaurant[:name] == "Express" || restaurant[:name] == "Kårrestaurangen"
```

Comment:
Every resturant doesn't supply this information and checking for the images would just make it seem to not contain any allergens instead which is a problem